Main.html

```diff
@@ -1,28 +1,36 @@
-<head> </head>
 
-<body>
+<html>
+<head> 
   <link href="Elements.css" rel="stylesheet" />
+  <title>Александр Валерьевич</title>
+</head>
+
+<body>
 
   <div id="outside_container"></div>
 
   <div id="container">
     <div id="menu_container">
-      <ul id="menu">
-        <li><a href="#container">Главная</a></li>
-        <li><a href="#rez">Резюме</a></li>
-        <li><a href="#cont">Контакты</a></li>
-        <li><a href="">Продам гараж</a></li>
-      </ul>
+      <nav>
+        <ul id="menu">
+          <li><a href="#container">Главная</a></li>
+          <li><a href="#rez">Резюме</a></li>
+          <li><a href="#cont">Контакты</a></li>
+          <li><a href="">Продам гараж</a></li>
+        </ul>
+      </nav>
     </div>
     <div id="avatar_container">
-      <div id="photo"></div>
+      <div id="photo">
+        <img src="photo.jpeg" alt="Фото" style = "height: 100%; width: 100%; border-radius: 50%" >
```

Comment:
When it's possible don't use inline-styles for your elements.
Move them to css.